db/migrate/20170207192912_encrypt_otp_secret.rb

```diff
@@ -0,0 +1,22 @@
+class EncryptOtpSecret < ActiveRecord::Migration
+  class User < ActiveRecord::Base
+  end
+
+  def up
+    encrypt_otp_secret_key
+  end
+
+  def down
+  end
+
+  def encrypt_otp_secret_key
+    User.where.not(otp_secret_key: nil).find_in_batches.with_index do |users, batch|
+      users.each do |user|
+        encrypted_attribute = EncryptedAttribute.new_from_decrypted(user.otp_secret_key)
```

Comment:
Yes, I think the idea is that we'd run the one-off script today and it would work. In the future, when migrations run, we wouldn't reference the class so migrating the database wouldn't fail.